priv/static/static/main.css

```diff
@@ -111,3 +111,8 @@ body {
   padding-right: 1.5em;
   text-align: right;
 }
+
+.snippet {
+  cursor: default;
+  background: transparent;
```

Comment:
When using `readonly`, the input gets a nasty, gray background. Setting the background to `transparent` sets it back to normal.